packages/nx/src/commands-runner/create-command-graph.ts

```diff
@@ -0,0 +1,51 @@
+import { ProjectGraph } from '../config/project-graph';
+import { NxArgs } from '../utils/command-line-utils';
+import { output } from '../utils/output';
+import { findCycle, makeAcyclic } from './command-graph-utils';
+
+export function createCommandGraph(
+  projectGraph: ProjectGraph,
+  projectNames: string[],
+  nxArgs: NxArgs
+) {
+  const dependencies: Record<string, string[]> = {};
+  for (const projectName of projectNames) {
+    if (projectGraph.dependencies[projectName].length >= 1) {
+      dependencies[projectName] = [
+        ...new Set(
+          projectGraph.dependencies[projectName].map(
+            (projectDep) => projectDep.target
+          )
```

Comment:
Filter out projects that are not workspace nodes because they don't have commands associated with them.